### DISCUSSION TITLE
how to remove specific clusters from UMAP

### DISCUSSION THREAD

**Discussion by YayanFeng55** [GreenleafLab/ArchR#1973] [Questions / Documentation] [answered]
Dear all,

Who knows how to remove specific clusters from UMAP? 

Thanks
Yayan

**Comment by YayanFeng55** [comment, marked as answer]
Solved by myself. Hope below code will help you guys with similar issue:

idxSample <- BiocGenerics::which(projMulti2$Clusters_Combined %ni% c("C2","C3","C12"))
cellsSample <- projMulti2$Clusters_Combined[idxSample]
projMulti3 <- projMulti2[projMulti2@cellColData$Clusters_Combined %in% cellsSample]

Thanks
Yayan

**Comment by Jyoung-0108** [comment]
hello, Thanks for your advice, I wonder that do i need to rerun 'addIterativeLSI', 'addCluster', and 'addUMAP' after i remove the unwanted cell cluster?
